### PR TITLE
Add prefect-aws recipe

### PR DESCRIPTION
**Pull request by skrsna**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dbast** [comment]
Looks good... maybe somebody from the prefect team wants to co-maintain, Pinging @conda-forge/prefect 

It is even possible to add the group `@conda-forge/prefect` as maintainer, if that is desired.

**Comment by skrsna** [comment]
> Looks good... maybe somebody from the prefect team wants to co-maintain, Pinging @conda-forge/prefect
> 
> It is even possible to add the group `@conda-forge/prefect` as maintainer, if that is desired.

Thanks for the review, I believe the windows build is failing due to an illegal character in the `README`.

Happy to co-maintain this feedstock with @conda-forge/prefect team :)

**Comment by madkinsz** [comment]
Thanks for opening this! I'm not sure what our standard here is. cc @ahuang11

**Comment by ahuang11** [comment]
I'd volunteer to be added to the maintainer's list.

Also, I think it's because of the star in the README. Maybe we can remove it in the next release if necessary?

File: recipes/prefect-aws/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  license_file: LICENSE
+  license_family: Apache
+  summary: Prefect collection of tasks and subflows to integrate with AWS
+  doc_url: https://prefecthq.github.io/prefect-aws/
+  dev_url: https://github.com/PrefectHQ/prefect-aws
+  
+
+extra:
+  recipe-maintainers:
+    - skrsna
```
Comment (skrsna) [inline recipes/prefect-aws/meta.yaml:47]:
```suggestion
    - skrsna
    - ahuang11
```

**Comment by skrsna** [comment]
> I'd volunteer to be added to the maintainer's list.

Thanks, added you as a maintainer https://github.com/conda-forge/staged-recipes/pull/21583/commits/95fdc38b79ce999331c7af4ca7389fa43998fa01

> Also, I think it's because of the star in the README. Maybe we can remove it in the next release if necessary?

Interesting, looks like it worked for `prefect-dask` in this PR https://github.com/conda-forge/staged-recipes/pull/20366 with the ⭐

**Comment by desertaxle** [comment]
Thanks for opening this @skrsna! Since `prefect-aws` is a Prefect-maintained library and @ahuang11 and myself are the primary maintainers of `prefect-aws`, my request is that the two of us would be listed as the recipe maintainers.

File: recipes/prefect-aws/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - ahuang11
```
Comment (skrsna) [inline recipes/prefect-aws/meta.yaml:48]:
```suggestion
    - ahuang11
    - desertaxle
```

**Comment by skrsna** [comment]
> Thanks for opening this @skrsna! Since `prefect-aws` is a Prefect-maintained library and @ahuang11 and myself are the primary maintainers of `prefect-aws`, my request is that the two of us would be listed as the recipe maintainers.

My bad, I didn't know who to add as maintainers earlier, added you to the maintainer's list https://github.com/conda-forge/staged-recipes/pull/21583/commits/a4516b2cee0ca5a4a7e64c93a7ed45f1df9d323b

**Comment by skrsna** [comment]
@conda-forge-admin, please ping conda-forge/help-python.

 Can we please skip the win64 build for now, I believe we need `encoding="utf-8"` [here](https://github.com/PrefectHQ/prefect-aws/blob/v0.2.1/setup.py#L11), but I could be wrong as well cause I don't have lot of experience with windows builds

**Comment by dbast** [comment]
the package is noarch, so the build will be done on linux after the feedstock conversion.

**Pull request merged**